app-tracking-protection/vpn-impl/src/main/res/values/strings-vpn.xml

```diff
@@ -242,7 +245,8 @@
         <item quantity="one">App</item>
         <item quantity="other">Apps</item>
     </plurals>
-    <string name="atp_ActivityEnabledLabel">This feature is in beta. <annotation type="app_settings_link">Having problems with an app?</annotation></string>
+    <!-- TODO [ANA] Modified, double check if new key is needed -->
+    <string name="atp_ActivityEnabledLabel">During beta, protection for some apps is automatically disabled. <annotation type="app_settings_link">View Apps</annotation></string>
```

Comment:
ditto, seems like a . is missing?

Reply:
I checked the final copy and it doesn't have a `.` at the end.